.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: BUG
```

Comment:
would it be worth it to have a label like "not verified" to indicate that the bug hasn't been checked yet by a core team developer? as the tickets don't move automatically to project and "user feedback" lane this would make them a bit more obvious to spot

Reply:
Yeah it depends what we want to do. We could discuss this on Monday. Ideally they would be put directly to User Feedback or so.

Reply:
I created `Needs Triage`